modules/gdscript/gdscript_compiler.cpp

```diff
@@ -232,6 +232,8 @@ static bool _is_exact_type(const PropertyInfo &p_par_type, const GDScriptDataTyp
 }
 
 static bool _can_use_validate_call(const MethodBind *p_method, const Vector<GDScriptCodeGenerator::Address> &p_arguments) {
+	ERR_FAIL_COND_V(p_method, false);
```

Comment:
```suggestion
	ERR_FAIL_NULL_V(p_method, false);
```
You used the wrong check